presto-cli/pom.xml

```diff
@@ -170,6 +183,96 @@
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <name>${project.name}</name>
+                    <version>${project.version}</version>
+                    <release>1</release>
+                    <group>Applications/Databases</group>
+                    <description>Presto CLI RPM Package.</description>
+                    <needarch>x86_64</needarch>
+                    <copyTo>${project.build.directory}/rpm/${project.build.finalName}.x86_64.rpm</copyTo>
+
+                    <mappings>
+                        <mapping>
+                            <directory>/usr/local/bin</directory>
+                            <filemode>755</filemode>
+                            <sources>
+                                <source>
+                                    <location>target/presto-cli-${project.version}-executable.jar</location>
+                                    <destination>presto-cli</destination>
+                                </source>
+                            </sources>
+                        </mapping>
+                    </mappings>
+                </configuration>
+            </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>enforce-file-size</id>
+                        <phase>verify</phase>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <requireFilesSize>
+                                    <!-- Maven Central has a 1GB limit -->
+                                    <maxsize>1106000000</maxsize>
+                                    <files>
+                                        <file>${project.build.directory}/rpm/${project.build.finalName}.x86_64.rpm</file>
+                                    </files>
+                                </requireFilesSize>
+                            </rules>
+                            <fail>true</fail>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <profile>
+            <id>ci</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <version>2.22.2</version>
+                        <configuration>
+                            <systemPropertyVariables>
+                                <rpm>${project.build.directory}/rpm/${project.build.finalName}.x86_64.rpm</rpm>
```

Comment:
Should the value for this property be defined in the pom properties? This will make it DRY and less chances of error since this is also repeated at https://github.com/prestosql/presto/pull/3958/files#diff-ac6e696e3482624c99ecfbf075a5b922R206 and https://github.com/prestosql/presto/pull/3958/files#diff-ac6e696e3482624c99ecfbf075a5b922R239